polaris-service/src/main/java/io/polaris/service/catalog/PolarisCatalogHandlerWrapper.java

```diff
@@ -578,7 +578,7 @@ public LoadTableResponse createTableDirectWithWriteDelegation(
                   .withLocation(request.location())
                   .withPartitionSpec(request.spec())
                   .withSortOrder(request.writeOrder())
-                  .withProperties(request.properties())
+                  .withProperties(properties)
```

Comment:
Looks like this change doesn't fit into the description/title of this PR. Mind moving this change to a separate PR?

Reply:
Separated into https://github.com/apache/polaris/pull/207